packages/internals/src/utils/loadEnvFile.ts

```diff
@@ -1,14 +1,29 @@
+import { PrismaConfig } from '@prisma/config'
+import Debug from '@prisma/debug'
+
 import { getEnvPaths } from './getEnvPaths'
 import { tryLoadEnvs } from './tryLoadEnvs'
 
+const debug = Debug('prisma:cli:loadEnvFile')
+
 /**
  * Read .env file only if next to schema.prisma
- * .env found: print to console it's relative path
+ * .env found: print to console its relative path
  */
 export async function loadEnvFile({
   schemaPath,
+  config,
   printMessage = false,
-}: { schemaPath?: string; printMessage?: boolean } = {}) {
+}: {
+  schemaPath?: string
+  printMessage?: boolean
+  config: PrismaConfig
+}) {
+  if (config.loadedFromFile) {
+    debug(`Prisma config detected, skipping environment variable loading`)
```

Comment:
We should probably write this to user in normal `stdout`, with the same format that we use to tell them that "Prisma has loaded the env vars for them" in the other conditional branch.

Reply:
Addressed in https://github.com/prisma/prisma/pull/26315